examples/seismic/acoustic/acoustic_example.py

```diff
@@ -99,4 +99,4 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 
 
 if __name__ == "__main__":
-    run(full_run=False, autotune=False, space_order=6, time_order=2)
+    run(full_run=True, autotune=False, space_order=6, time_order=2)
```

Comment:
:grimacing: `full_run=True` tends to completely kill laptops with only 4GB mem :cry:, so it's kind of an evil landmine to have lying around, especially with no killswitch.  I'd much prefer some basic option processing with safe defaults (eg. `python acoustic_example.py --full -so 12 --autotune` or similar). It's fine for this PR, but worth considering in one of the next ones, because I will eventually delete this test in a fit of rage if it takes out my machine again. :smiling_imp: